test-app/ios-uikit/TestApp/TestAppViewController.swift

```diff
@@ -42,7 +40,15 @@ class TestAppViewController : UIViewController {
             codeListener: CodeListener()
         )
         ExposedKt.bindWhenReady(content: content, view: treehouseView)
-        view = treehouseView.view
+
+        let tv = treehouseView.view
+        tv.translatesAutoresizingMaskIntoConstraints = false
+
+        self.view.addSubview(tv)
+        let safeGuide = self.view.safeAreaLayoutGuide
```

Comment:
Oh sweet. Will try.

Reply:
If this works, it's worth pointing out that it's a bit backwards/unconventional--a `UIView`'s `bounds` rect is expressed relative to the view's own coordinate system, whereas its `frame` rect is expressed relative to its `superview`. It's more typical to declare `mySubview.frame = mySuperview.bounds`, to get `mySubview` to fill `mySuperview`'s rect.

In this case, `view.safeAreaLayoutGuide.layoutFrame` is expressed `view`'s coordinate system, not `tv`'s.

Reply:
Looks like they're not exactly in the same coordinate system and the bound origin or something gets skewed by half the height/width. Even when I fix it (or try to) it occupies the whole screen still rather than only the safe area.

![Screenshot 2023-09-25 at 8 14 55 PM](https://github.com/cashapp/redwood/assets/66577/e9de803b-d7be-47bb-99b4-2502c4489780)

Reply:
Going to merge as-is. Hopefully soon we'll have a scaffolding-like composable in this app which gets us a traditional top/nav bar on Android and iOS.

Reply:
Good to know--it was worth a shot. Definitely safe to merge as is!